src/Dapr.Actors.AspNetCore/WebHostBuilderExtensions.cs

```diff
@@ -53,9 +57,18 @@ public static IWebHostBuilder UseActors(this IWebHostBuilder hostBuilder, Action
                 services.AddHealthChecks();
                 services.AddSingleton<IStartupFilter>(new DaprActorSetupFilter());
 
-                services.AddSingleton<ActorRuntime>(runtime);
+                services.AddSingleton<ActorRuntime>(s =>
+                {
+                    return new ActorRuntime(s.GetRequiredService<IOptions<DaprActorRuntimeOptions>>().Value, s.GetRequiredService<ILoggerFactory>());
+                });
             });
 
+            hostBuilder.ConfigureAppConfiguration((hostingContext, config) =>
+            {
+                config.AddJsonFile("appsettings.json", 
+                    optional: true, 
+                    reloadOnChange: true);
+            });
```

Comment:
This should not be here, Dapr should not have an opinion about what configuration files you use - this is already part of most projects without us having any code for it.

Reply:
one of my trial changes. removed